### PR TITLE
update details of project

### PR DESCRIPTION
**Pull request by tanaypratap**
Added my projects section which will list the apps I am developing in neog.camp

### CODE REVIEW

**Comment by tanaypratap** [comment]
PR looks good to merge. Go ahead.

**Comment by Rudrahari** [comment]
> PR looks good to merge. Go ahead.

nice

**Comment by nivet79** [comment]
Good work tanay.

**Comment by singh3069** [comment]
sir pr looks good

**Comment by Kaustubh-Dengle** [comment]
cool

**Comment by utsavkumar-280** [comment]
looking fine,merge it

**Comment by dhwajsharma** [comment]
Looks kinda nice

**Comment by ar-aaradhya** [comment]
It looks good.

**Comment by tonnyvijayan** [comment]
amazing!!!

**Comment by jainlokesh318** [comment]
Great

**Comment by SauravKumar-max** [comment]
Tanay its awesome.

**Comment by aadhilkhan** [comment]
Looks good to go.

**Comment by PriyanshuSinghR** [comment]
pretty good

**Comment by sachin0196** [comment]
the changes look ready for deployement.

amazing.

**Comment by SobhanDash** [comment]
Looks great Tanay!

**Comment by Akshaay9** [comment]
Tanay, this looks good, u can proceed....

**Comment by Garima-sharma814** [comment]
Happy learning !!

**Comment by Saurabh-2412** [comment]
Awesome..!fab..

**Comment by vikasganiga05** [comment]
# Thank You Tanay Sir :heart:

**Comment by suyashpradhan** [comment]
Now I understood what is PR, Looks Good Thanks!!

**Comment by soumitdas** [comment]
Hey @tanaypratap It's cool

**Comment by aritrodhar** [comment]
Great Effort, Sir

**Comment by ar-aaradhya** [comment]
good

**Comment by ramji-ram** [comment]
works fine

**Comment by thunderfreak** [comment]
understanding well sir

**Comment by aditya6521** [comment]
Good job... merge it.

**Comment by rushi-173** [comment]
All good.

**Comment by akashkushwaha02** [comment]
Niceeee

**Comment by srividyamvenkatesh** [comment]
Great going!

**Comment by Dtech-Dbug** [comment]
try Block instead of writing everything inline. GRreat work.

**Comment by nitindang** [comment]
Howdy Tanay ???

**Comment by kalpana-7** [comment]
Great go ahead 👍

**Comment by RBOSAMIYA9** [comment]
This looks good 👍 you should merge it.

**Comment by silent-archary** [comment]
હા મોજ હા

**Comment by jainPrajwal** [comment]
Looks great!!

**Comment by silent-archary** [comment]
હા મારા રોયા

**Comment by AnkitaBagale** [comment]
yuhuu!!!!

**Comment by jay-rajput30** [comment]
thank you

**Comment by shivammishra-6** [comment]
Great going, Sir! Ready to merge now.

**Comment by singhxamit21** [comment]
Sir it looks good

**Comment by harshitbadollacodes** [comment]
Looks good.

**Comment by ArnavRaaj** [comment]
better late than never. Thank you, sir :)

**Comment by anurag4512** [comment]
It looks good sir! merge it.

**Comment by Mohammad-Naved** [comment]
It Looks good.

**Comment by abhishekajs** [comment]
It looks good

**Comment by saisumanthkacham** [comment]
amazing , go a head

**Comment by RadhikaRJ** [comment]
PR is great to go ahead with.

**Comment by ranjitm10** [comment]
Cool sir

**Comment by imaxisXD** [comment]
pr is looking good, go ahead and merge it

**Comment by dev-ameyjoshi** [comment]
Pull request is great, completed lect-3 as well,onto LEC-4 now.

**Comment by ShasheeshPurohit** [comment]
PR looks good to be merged.

**Comment by DevMittal04** [comment]
PR looks pretty good EVEN NOW!

**Comment by Souvik48** [comment]
PR looks dope!

**Comment by samarthkhurana** [comment]
PR looks good to merge sir. Please go ahead

**Comment by arpit-yadv** [comment]
PR is good to go .

**Comment by Abhay5855** [comment]
Its look good

**Comment by akashlisp** [comment]
good to go. but where do i find preview??

**Comment by PandaArchives** [comment]
PR looks good to merge

**Comment by Atharva-77** [comment]
nice

**Comment by piyushsingh0992** [comment]
this looks good

**Comment by zishan-ansari** [comment]
this looks good

**Comment by poonammohata** [comment]
nice work

**Comment by shashank1223** [comment]
PR looks good to merge. Go ahead.

**Comment by simrananand-2-11** [comment]
PR looks clean.

**Comment by ckahub** [comment]
PR looks good to merge. Go ahead

**Comment by RohitKota3** [comment]
Learning: Good to merge! Go ahead!

**Comment by Shahid-786** [comment]
Good work tanay. GO ahead !

**Comment by anushree71199** [comment]
Great work Tanay

**Comment by KhushaliSamderiya** [comment]
Good to merge!

**Comment by rajatyadav26** [comment]
PR looks great . Good to go.

**Comment by Abhirup-Sarkar** [comment]
Go ahead

**Comment by Tiwarishashwat** [comment]
looks good!

**Comment by chandana105** [comment]
_It looks good!_

**Comment by anshumanjaiswal** [comment]
Nice updates!

**Comment by avadheshchauhan** [comment]
done, Commenting.

**Comment by Manju-sree** [comment]
You are like a angel sent by god to learn web dev to us , thanks a ton :)

**Comment by yamsun** [comment]
great

**Comment by ashwindevx** [comment]
!!

**Comment by ghost** [comment]
PR looks good to merge.

**Comment by devang-stack** [comment]
nice

**Comment by kesarivenkatesh** [comment]
Branches and PRs are nice feature to have

**Comment by Aman1440** [comment]
practising a bit late... pr always look good when it's coming from u, sir!!

**Comment by MiheerTamkhane** [comment]
boht hard tanay sir. You are greate. Thank you.

**Comment by TusharSNagpal** [comment]
Cool. You are doing good work for students like me.

**Comment by prakash855** [comment]
PR to chhoro tanay bhaiya all you are teaching is best

**Comment by piyushsinghhsv** [comment]
pr looks good

**Comment by Sen-442b** [comment]
I'm so late but yeah , good to go

**Comment by rudratalur** [comment]
Ohh!! now i understood PR. go ahead..
Thanks!

**Comment by mohannagabhiru** [comment]
good to merge now

**Comment by chunchuakhil** [comment]
PR:   good to merge now

**Comment by way2nizam** [comment]
I'm late but good to go

**Comment by Nikhil-Kaul** [comment]
Good merge!

**Comment by mkcconnect** [comment]
PR is fine. Proceed to merge.

**Comment by starver20** [comment]
Good to go!

**Comment by mdJawwad96** [comment]
It's good to go

**Comment by aditi2802** [comment]
> Added my projects section which will list the apps I am developing in neog.camp
A BIG THANKYOU :)

**Comment by SAURABHSIN996** [comment]
pr looks good ,go ahead

**Comment by voradhwani** [comment]
PR looks good.

**Comment by shambhavisud** [comment]
looks goof

**Comment by shambhavisud** [comment]
*good

**Comment by swapilverma** [comment]
Looks preety good to me.

**Comment by sarb001** [comment]
Hey! Tanay U r doing Great 
Thanks 
Pull request  Done

**Comment by Srinathyadav21** [comment]
good

**Comment by ankit25-x** [comment]
looks good sir

**Comment by AshokRJadhav** [comment]
PR Looks perfect

**Comment by poornachary1998** [comment]
Hey! Tanay.

It's Good

**Comment by shivammishra-6** [comment]
**Great**

**Comment by chethana-crypto** [comment]
PR looks fine...so ur good to go further

**Comment by HarshitaGupta16** [comment]
PR looks perfect. You are good to go!

**Comment by anurag-kkumarr** [comment]
PR looks good to go.

**Comment by Shah-Arsalan** [comment]
You are really helping people out here.

**Comment by debojyotiroy245** [comment]
No conflicts with the base branch. PR looks good.

**Review by ghost — COMMENTED** [review]
Nice Work Tanay !

**Comment by chetannikam** [comment]
PR looks good Go Ahead

**Comment by paul-coder-22** [comment]
It's looks **good**

**Comment by harshitadaswani** [comment]
PR looks great. You can merge it.

**Comment by yogesh-** [comment]
Looks great.

**Comment by aqsaaqeel** [comment]
Looks good

**Comment by govardhanreddyc79** [comment]
Looks good

**Comment by Abhishek-295** [comment]
great work sir

**Comment by Divya70** [comment]
Its really looks good.

**Comment by JuvitaSaini09** [comment]
it looks good to merge

**Comment by NagaSatyaGeeth** [comment]
It looks nice! Deploy it Tanay

**Comment by VinayakaHegade** [comment]
looks good. Tanay you can merge.

**Comment by Developers-codz** [comment]
Looks good Tanay sir

**Comment by Prathish-codes** [comment]
Looks cool

**Comment by Kavita539** [comment]
It looks good

**Comment by ghost** [comment]
Looks good

**Comment by krishnakant01** [comment]
looks good sir!

**Comment by Bhakti199** [comment]
It looks good Tanay.

**Comment by PiotrD94** [comment]
PR looks good to merge. Go ahead. Thanks

**Comment by atmanishk** [comment]
It looks cool, Sir.

**Comment by dubeyaditya29** [comment]
PR looks good ready to merge to main!

**Comment by dwivedialind** [comment]
pr looks good

**Comment by Anurag021** [comment]
PR looks very good, please go ahead.

**Comment by tanishk15sharma** [comment]
great work !

**Comment by ianuragsharma** [comment]
Noice

**Comment by abhishekParashar-sudo** [comment]
kem palty

**Comment by krgarima** [comment]
PR looks good!

**Comment by Veekshith-Thoram** [comment]
Tanay this looks good

**Comment by Maianki** [comment]
Looks good Tanay.

**Comment by himadri2110** [comment]
Good work, Tanay

**Comment by RobinKartikeyaKhatri** [comment]
PR looks very good, please go ahead.

**Comment by NikunjMalani3** [comment]
Looks good. Good to go

**Comment by mkcconnect** [comment]
Please merge.

**Comment by aashikofsach** [comment]
Its right

**Comment by bhavyaj12** [comment]
Looks great. Good to merge!

**Comment by archit7448** [comment]
looks great . merge it

**Comment by imrohan997** [comment]
It looks good Tanay

**Comment by rahulyadav139** [comment]
looks great!

**Comment by sunnythechamp** [comment]
it's look goods

**Comment by ishant005** [comment]
looks good

**Comment by krmukesh1** [comment]
looks goos

**Comment by Ramkonda31** [comment]
PR looks good

**Comment by amanchauhann** [comment]
PR is fine Tanay, you can go ahead. Will see you in office tommorow.

**Comment by ManpreetSingh18** [comment]
Thanks for your work

**Comment by ghost** [comment]
Looks good

**Comment by vaibhav96t6** [comment]
all set to go!

**Comment by mohak0011** [comment]
Looks Good, go ahead

**Comment by aman225** [comment]
we love your site

**Comment by KhandelwalBhumika** [comment]
This works. Go ahead!

**Comment by shailajaBegari** [comment]
Nice sir.

**Comment by anshulshuks** [comment]
A year late, but I am learning now!

**Comment by AdityaGauranga** [comment]
The PR looks good to merge :)

**Review by keshav260 — COMMENTED** [review]
cool

**Comment by Dorababupc** [comment]
good

**Comment by RakshitB07** [comment]
Nice!

**Comment by Instinct29** [comment]
hmmm hmmm! revising for my level 1 interview. Merge request Pull request All good

**Comment by ofi5** [comment]
i am late.

**Comment by shivabaad** [comment]
Fantastic Tanay !

**Comment by krishnaprasad2201** [comment]
this is nice

**Comment by Bthakur12** [comment]
Looks Good Go ahead.

**Comment by HarshithaMuthyala** [comment]
Looks great. Go Ahead.

**Comment by Kalpana513** [comment]
LOOKS GOOD GO AHEAD!!!

**Comment by vishal9860** [comment]
good work

**Comment by antarikshJn** [comment]
good one

**Comment by PraneethaCharugundla** [comment]
looks good

**Comment by rohitmane8356** [comment]
> Added my projects section which will list the apps I am developing in neog.camp

nice

**Comment by ArchitBorde** [comment]
This looks good

**Comment by AravindGurav** [comment]
The PR looks good, you can merge it now

**Comment by codekundan** [comment]
found Tanay channel late. It looks good Tanay.

**Comment by adhairyaa** [comment]
looks Good

**Comment by belwalshubham** [comment]
PR looks good. Go ahead

**Review by Kavryo — COMMENTED** [review]
Looks Good!

**Review by chandykozo — COMMENTED** [review]
feels good man. 😊

**Comment by ahirraokartik** [comment]
GG

**Comment by alamusneha** [comment]
good to go

**Comment by mitcareem** [comment]
awesome work!

**Review by kaushl3ndra — COMMENTED** [review]
PR looks good to merge. Go ahead.

**Comment by iv-krish** [comment]
hey i am learning

**Comment by scam1992** [comment]
PR looks good....

**Comment by harshtrive** [comment]
looks good

**Comment by MihirD77** [comment]
PR looks Awesome!

**Comment by yaseenmalik008** [comment]
PR looks good!

**Comment by Anush79** [comment]
I am impressed:)

**Comment by hasrshtrivedi** [comment]
awesome!!

**Comment by rudrakshithakur** [comment]
Looks good!!

**Comment by asvarade** [comment]
looks good. go ahead....

**Comment by shivaniss24** [comment]
It looks really good!. Thank you tanay sir

**Comment by unnati369** [comment]
looks good.

**Comment by akshadjaiswal** [comment]
looks good. ready to merge it

**Comment by Vedant005** [comment]
it really looks good.

**Comment by divyaarora1808** [comment]
The PR looks great ! Lets merge it now!

**Comment by divyaarora1808** [comment]
# Awesome

**Comment by ProgrammerAvinash** [comment]
nice work

**Comment by iamtoufiq** [comment]
perfect

**Comment by insayed** [comment]
Looks good! Good to go.

**Comment by aaliyanafis** [comment]
Hey Tanay! PR looks Great😊

**Comment by gopala171** [comment]
It looks good!

**Comment by sumit2060049** [comment]
its looks good tanay sir.

**Comment by TestaniruddhaGoswami** [comment]
this looks good, merge the PR. Go ahead.

**Comment by AmeyaP29** [comment]
Looks good. Merge the PR

**Comment by Ttkode** [comment]
Good

**Comment by anshsgit** [comment]
> Added my projects section which will list the apps I am developing in neog.camp

Good yeah

**Comment by anshsgit** [comment]
Good yeah

**Comment by hritik2268** [comment]
> Added my projects section which will list the apps I am developing in neog.camp

 watching ur video now , it actually looks good

**Comment by Stefin-Abraham** [comment]
Great Work Tanay

**Comment by hsihsa8789** [comment]
Thank you sir

**Comment by ma-sadiq** [comment]
PR looks good.... go ahead....!

**Comment by YuvrajPathania** [comment]
it's look u can merge it

**Comment by dassoumen98** [comment]
its good PR

**Comment by Suraj5260** [comment]
Looks good. Go on.

**Comment by Stefin-Abraham** [comment]
Great SIr

**Comment by AlaHariVital** [comment]
good

File: index.html [hunk 1]
```diff
@@ -10,6 +10,8 @@
 <body>
   Hi! I am Tanay Pratap!
     I work at Microsoft.
+
```
Comment (PratikKanthe) [inline index.html:13]: it looks good

**Review by PratikKanthe — COMMENTED** [review]
cool

**Comment by IamSinghYuvraj** [comment]
pr looks good

**Comment by SunithaPatil** [comment]
looks good, go ahed

**Comment by garimagiri** [comment]
PR looks good

**Comment by arjun26122002** [comment]
nice

**Comment by ARSHIKULSUM** [comment]
thank you sir.

**Comment by samuelannepogu** [comment]
That looks fine sir.

File: index.html (hunk 1 above)
Comment (pr0698) [reply index.html:13]: it looks good

**Review by Arathy23 — COMMENTED** [review]
looks good ! proceed to merge

**Comment by pranjaljaiswal10** [comment]
Awesome

**Comment by ghost** [comment]
Looks Good!!

**Comment by Ritik240910** [comment]
PR looks great

**Comment by SushantSardar** [comment]
Pr looks great even after 2 years..........he he he

**Comment by 111shaikafzal** [comment]
PR looking code proceed

**Comment by cssprasanth99** [comment]
looks good

**Comment by Sonal121992** [comment]
It looks good Tanay

**Comment by yashas8197** [comment]
its really good

**Comment by Shivani-0503** [comment]
Looks Good

**Comment by saranyasivarajcse** [comment]
PR looks good.. Go ahead.

**Comment by neerajkumar002** [comment]
PR looks good.. sir

**Comment by chinz14** [comment]
tanay sir, the PR looks good:)

**Comment by shivangi-sik** [comment]
Looks awesome

**Comment by yashru17** [comment]
Thank you so much tanay Bhai

**Comment by webwithaman** [comment]
Aspiration to join neogcamp 2024
Hope Got it.

**Comment by Vaibhu117** [comment]
Niceee

**Comment by sumitsingh43** [comment]
hello sir i am learning with you thank you

**Comment by MAURICEDS0UZA** [comment]
PR looks good !

**Comment by Trisha4041** [comment]
Wow!!!Its awesome

**Comment by krishna-9-deepali** [comment]
It looks good .

**Comment by Messionat45** [comment]
Hi

**Review by ehsanA380 — APPROVED** [review]

**Comment by TanishaKhatri** [comment]
Thank you

**Pull request merged**